test/base_test.py

```diff
@@ -364,7 +364,7 @@ def test_meta_on_broken_soft_link(self, server, resolve_links):
             }
 
     def test_403_on_file_without_read_permission(self, server):
-        filename = "test.h5"
+        filename = "test_permission.h5"
```

Comment:
I renamed this file since it could prevent from using `test.h5` as name for the file in other tests due to the reading permission missing.